CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 2.8.3 (2020-10-07)
+
+### Breaking changes
```

Comment:
when you have breaking changes you should increment minor version, so here version should be 2.9.0.